greenswitch/esl.py

```diff
@@ -476,7 +481,6 @@ def listen(self):
             if self.connection_count >= self.max_connections:
                 logging.info('Rejecting call, server is at full capacity, current connection count is %s/%s' %
                              (self.connection_count, self.max_connections))
-                gevent.sleep(0.1)
                 session.stop()
```

Comment:
the channel stays stuck in the socket application if sent an "exit" command before of "connect".

```
54efeb8a-a097-11e8-9114-c7e85da8e7aa 2018-08-15 11:27:26.273460 [DEBUG] mod_dptools.c:1693 UNSET [force_transfer_context]
54efeb8a-a097-11e8-9114-c7e85da8e7aa EXECUTE sofia/internal/600@edev socket(127.0.0.1:5000 async full)
54efeb8a-a097-11e8-9114-c7e85da8e7aa 2018-08-15 11:27:26.273460 [NOTICE] mod_event_socket.c:447 Trying host: 127.0.0.1:5000

freeswitch@evangilo-op> show channels
uuid,direction,created,created_epoch,name,state,cid_name,cid_num,ip_addr,dest,application,application_data,dialplan,context,read_codec,read_rate,read_bit_rate,write_codec,write_rate,write_bit_rate,secure,hostname,presence_id,presence_data,accountcode,callstate,callee_name,callee_num,callee_direction,call_uuid,sent_callee_name,sent_callee_num,initial_cid_name,initial_cid_num,initial_ip_addr,initial_dest,initial_dialplan,initial_context
54efeb8a-a097-11e8-9114-c7e85da8e7aa,inbound,2018-08-15 11:27:26,1534343246,sofia/internal/600@edev,CS_EXECUTE,600,600,192.168.202.1,1010,socket,127.0.0.1:5000 async full,XML,out-extensions,,,,,,,,evangilo-op,600@edev,,dev,RINGING,,,,,,,600,600,192.168.202.1,1010,XML,out-extensions

1 total.

```